docs/source/guide/explanation/algorithms/classification/multi_label_classification.rst

```diff
@@ -52,11 +52,47 @@ In the table below the `mAP <https://en.wikipedia.org/w/index.php?title=Informat
 | EfficientNet-V2-S     | 91.91           | 77.28     | 71.52            | 80.24     |
 +-----------------------+-----------------+-----------+------------------+-----------+
 
-.. ************************
-.. Semi-supervised Learning
-.. ************************
+************************
+Semi-supervised Learning
+************************
 
-.. To be added soon
+Semi-SL (Semi-supervised Learning) is a type of machine learning algorithm that uses both labeled and unlabeled data to improve the performance of the model. This is particularly useful when labeled data is limited, expensive or time-consuming to obtain.
+
+To utilize unlabeled data during training, we use `BarlowTwins loss <https://arxiv.org/abs/2103.03230>`_ as an auxiliary loss for Semi-SL task solving. BarlowTwins enforces consistency across augmented versions of the same data (both labeled and unlabeled). Additionally, our algorithm uses a combination of strong data augmentations and a specific optimizer called Sharpness-Aware Minimization (SAM) to further improve the accuracy of the model.
+
+Overall, OpenVINO™ Training Extensions utilizes powerful techniques for improving the performance of Semi-SL algorithm with limited labeled data. They can be particularly useful in domains where labeled data is expensive or difficult to obtain, and can help to reduce the time and cost associated with collecting labeled data.
+
+.. _mlc_cls_semi_supervised_pipeline:
+
+- ``BarlowTwins loss``: A specific implementation of Semi-SL that combines the use of a consistency loss with strong data augmentations, and a specific optimizer called Sharpness-Aware Minimization (SAM) to improve the performance of the model.
+
+- ``Adaptive loss auxiliary loss weighting``: A technique for assigning such a weight for an auxiliary loss that the resulting value is a predefined fraction of the EMA-smoothed main loss value. This method allows aligning contribution of the losses during different training phases.
+
+- ``Exponential Moving Average (EMA)``: A technique for maintaining a moving average of the model's parameters, which can improve the generalization performance of the model.
+
+- ``Additional techniques``: Other than that, we use several solutions that apply to supervised learning (No bias Decay, Augmentations, Early-Stopping, etc.)
```

Comment:
You also use "strong" and "weak" pipelines, could you describe them here?

Reply:
I described them in general description to clarify work of the BT loss.